manifests/f/FiloSottile/mkcert/1.4.4/FiloSottile.mkcert.defaultLocale.yml

```
@@ -0,0 +1,14 @@
+# yaml-language-server: $schema=https://github.com/microsoft/winget-cli/raw/master/schemas/JSON/manifests/v1.2.0/manifest.defaultLocale.1.2.0.json
+PackageIdentifier: FiloSottile.mkcert
+PackageVersion: 1.4.4
+PackageLocale: en-US
+Publisher: FiloSottile
+PublisherUrl: https://github.com/FiloSottile
+Author: Filippo Valsorda
+PackageName: mkcert
+PackageUrl: https://github.com/FiloSottile/mkcert
+License: BSD 3-Clause
+LicenseUrl: https://github.com/FiloSottile/mkcert/blob/master/LICENSE
+ShortDescription: mkcert is a simple tool for making locally-trusted development certificates.
+ManifestType: defaultLocale
+ManifestVersion: "1.2.0"
```

```suggestion
ManifestVersion: 1.2.0

```
Newline preferred at end of file